packages/main/test/pages/TimePicker.html

```diff
@@ -12,7 +12,11 @@
 
 		<script src="%VITE_BUNDLE_PATH%" type="module"></script>
 
-
+		<style>
+			[ui5-time-picker] {
```

Comment:
Make that selector with a class, inline-style or id for one component only, since now all of the dev samples (ui5-time-pickers) are with `height: 3 rem;`

Reply:
that was missed, it was only for test purposes. will be removed.